tests/integration/test_vulnerability_detector/test_feeds/data/configuration_template/configuration_cpe_indexing.yaml

```diff
@@ -0,0 +1,75 @@
+- sections:
+    - section: vulnerability-detector
+      elements:
+        - enabled:
+            value: 'yes'
+        - run_on_start:
+            value: 'yes'
+        - provider:
+            attributes:
+              - name: 'redhat'
+            elements:
+              - enabled:
+                  value: 'no'
+        - provider:
+            attributes:
+              - name: 'canonical'
+            elements:
+              - enabled:
+                  value: 'no'
+        - provider:
+            attributes:
+              - name: 'debian'
+            elements:
+              - enabled:
+                  value: 'no'
+        - provider:
+            attributes:
+              - name: 'msu'
+            elements:
+              - enabled:
+                  value: ENABLED
+              - path:
+                  value: MSU_JSON_PATH
+        - provider:
+            attributes:
+              - name: 'alas'
+            elements:
+              - enabled:
+                  value: 'no'
+        - provider:
+            attributes:
+              - name: 'arch'
+            elements:
+              - enabled:
+                  value: 'no'
+        - provider:
+            attributes:
+              - name: 'nvd'
+            elements:
+              - enabled:
+                  value: 'yes'
+              - path:
+                  value: NVD_JSON_PATH
+
+    - section: sca
+      elements:
+        - enabled:
+            value: 'no'
+
+    - section: rootcheck
+      elements:
+        - disabled:
+            value: 'yes'
+
+    - section: syscheck
+      elements:
+        - disabled:
+            value: 'yes'
+
+    - section: wodle
+      attributes:
+        - name: 'syscollector'
+      elements:
+        - disabled:
+            value: 'yes'
```

Comment:
Missing new line at the end. Review this in all files of the PR.

Reply:
Done in https://github.com/wazuh/wazuh/pull/17576/commits/733738ed093a36accd09961bf60064388b0d78f3